svg/attributes/events.json

```diff
@@ -37,6 +37,9 @@
                 "safari_ios": {
                   "version_added": null
                 },
+                "samsunginternet_android": {
+                  "version_added": null
```

Comment:
This whole file is null in every entry, which isn't great it may be worth testing it to see if we do support it.

Reply:
never mind almost all of SVG is reported as null. That would be a significant project to populate...

(Ada ponders writing tests for SVG, which render to canvas then read the pixels to determine the support for certain features)